packages/server/graphql/mutations/helpers/getReflectionSentimentScore.ts

```diff
@@ -0,0 +1,12 @@
+import getGoogleLanguageManager from '../../../getGoogleLanguageManager'
+import manageGoogleNLPErrorResponse from './manageGoogleNLPErrorResponse'
+
+const getReflectionSentimentScore = async (plaintextContent: string) => {
+  if (!plaintextContent) return 0.0
```

Comment:
-1 It makes sense for a quick prototype, but because we're storing these long term to present the data nicely, let's distinguish neutral score from no data